docs/src/content/docs/sdk.mdx

```diff
@@ -212,6 +212,25 @@ state.addEventListener('ultravoxExperimentalMessage', (msg) => {
 session.leaveCall();
 ```
 
+#### Debug Message:  Tool Call
```

Comment:
Can we add a warning against relying on the structure or content of these messages please?

Reply:
yessir